src/EPR.SubsidiaryBulkUpload.Application/Services/BulkUploadOrchestration.cs

```diff
@@ -25,13 +32,15 @@ public async Task Orchestrate(IEnumerable<CompaniesHouseCompany> data, Guid user
             async sg => (SubsidiaryGroup: sg, Org: await organisationService.GetCompanyByCompaniesHouseNumber(sg.Parent.companies_house_number)))
             .Where(sg => sg.Org != null);
 
-        await foreach(var subsidiaryGroupAndParentOrg in subsidiaryGroupsAndParentOrg)
+        await foreach (var subsidiaryGroupAndParentOrg in subsidiaryGroupsAndParentOrg)
         {
             await childProcessor.Process(
                 subsidiaryGroupAndParentOrg.SubsidiaryGroup.Subsidiaries,
                 subsidiaryGroupAndParentOrg.SubsidiaryGroup.Parent,
                 subsidiaryGroupAndParentOrg.Org,
                 userId);
```

Comment:
Use the user id from the user request model

Reply:
Updated.